src/components/button/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { FC, ReactNode } from 'react';
+import {
+  ButtonProps,
+  Button as CUButton,
+  StyleProps,
+  useTheme,
+  useToken,
+} from '@chakra-ui/react';
+
+interface Props extends ButtonProps {
+  children: ReactNode;
+  size?: 'md' | 'lg' | 'xl';
+  variant?: 'primary';
```

Comment:
I think that would be solid with Chakra https://chakra-ui.com/docs/components/form/button#button-variants

Reply:
Actually we're currently provided only 3 sizes of button, meanwhile, default Chakra provides 4. Them same situation with `variant`. That's a reason why I want to expose these options by our definition.